docs/error-codes.md

```diff
@@ -113,6 +113,18 @@ the error code. For example:
 
 #### `IL1030`: Invalid argument for '{token}' option
 
+#### `IL1031`: Invalid assembly action '{action}'
+
+#### `IL1032`: Trimming assembly '{assemblyFile}' could not be found
```

Comment:
Nit: I don't really understand the meaning of "trimming assembly" - does it mean the same as "assembly which is being trimmed"?

I guess I don't see why we need to prefix these with "Trimming" - we don't do that for the other error messages either...

Reply:
Changed